pinot-core/src/main/java/org/apache/pinot/core/transport/ServerRoutingInstance.java

```diff
@@ -23,43 +23,36 @@
 import java.util.Map;
 import java.util.concurrent.ConcurrentHashMap;
 import javax.annotation.concurrent.ThreadSafe;
-import org.apache.pinot.spi.config.table.TableType;
 import org.apache.pinot.spi.utils.CommonConstants.Helix;
 
 
 /**
  * The {@code ServerRoutingInstance} class represents the routing target instance which contains the information of
- * hostname, port, and table type it serves.
- * <p>Different table types on same host and port are counted as different instances. Therefore, one single Pinot Server
- * might be treated as two different routing target instances based on the types of table it serves.
+ * hostname and port.
  */
 @ThreadSafe
 public final class ServerRoutingInstance {
-  private static final String SHORT_OFFLINE_SUFFIX = "_O";
-  private static final String SHORT_REALTIME_SUFFIX = "_R";
```

Comment:
Instead of removing the `TableType`, is another option to **add** table name ? Then the rest of the code after `QueryRunner` would work as is since this code does not hard-code offline/realtime blocks ?